### PR TITLE
changed user information popup through vscode showInformationMessage

### PR DESCRIPTION
**Pull request by SachinS10-lab**
- Added chat model selection user information popup in configureAssistant.ts file.
- Added welcome user information popup in setupGranitePage.ts

-------

image

---

image

### CODE REVIEW

File: src/panels/setupGranitePage.ts [hunk 1]
```diff
@@ -230,6 +243,15 @@ export class SetupGranitePage {
...
         switch (command) {
           case "init":
+            // Check if we should show the Continue Onboarding message
+            const shouldShowContinueOnboarding = await this.shouldShowContinueOnboarding();
+            if (shouldShowContinueOnboarding) {
+              vscode.window.showInformationMessage(
+                `Upon launching Continue AI code Assistant, A Quickstart Popup will be displayed.
+                  Make sure to close it before setting up IBM Granite Code Models.`,
+                ``
+              );
```
Comment (fbricon) [inline src/panels/setupGranitePage.ts:253]: no, this message will only be shown once configuration is done

File: src/panels/setupGranitePage.ts [hunk 2]
```diff
@@ -212,6 +214,17 @@ export class SetupGranitePage {
...
+    const config = await readConfig(configFilePath);
+    if (!config || !config.models) {
+      vscode.window.showErrorMessage(`No Models Found!`);
+      return false;
+    }
+    return config.models.length === 1 &&
+      config.models[0].provider === "anthropic" &&
+      config.models[0].apiKey === "";
+  }
+
```
Comment (fbricon) [inline src/panels/setupGranitePage.ts:227]: please delete this

File: src/configureAssistant.ts [hunk 3]
```diff
@@ -103,8 +103,23 @@ export class AiAssistantConfigurator {
...
+      return false;
+    }
+    return config.models.length === 1 &&
+      config.models[0].provider === "anthropic" &&
+      config.models[0].apiKey === "";
+  }
+
   public async openWizard() {
     if (isContinueInstalled()) {
+      vscode.window.showInformationMessage(
```
Comment (fbricon) [inline src/configureAssistant.ts:118]: only display if mightShowContinueOnboarding returns true

File: src/configureAssistant.ts [hunk 4]
```diff
@@ -103,8 +103,23 @@ export class AiAssistantConfigurator {
     this.apiBase = DEFAULT_API_BASE;
   }
 
+  private async shouldShowContinueOnboarding(configFile: string): Promise<boolean> {
```
Comment (fbricon) [inline src/configureAssistant.ts:106]: rename to  mightShowContinueOnboarding

File: src/configureAssistant.ts (hunk 3 above)
Comment (fbricon) [reply src/configureAssistant.ts:118]: also, run this popup only after calling this.configureAssistant();

**Review by fbricon — CHANGES_REQUESTED** [review]

File: src/configureAssistant.ts (hunk 4 above)
Comment (SachinS10-lab) [reply src/configureAssistant.ts]: Changed all mentioned changes and now working as expected.


When mightShowContinueOnboarding become true.

image

--------

When mightShowContinueOnboarding become false.

image

**Comment by fbricon** [comment]
Thanks @SachinS10-lab

**Pull request merged**